openmetadata-ui/src/main/resources/ui/src/components/common/Form/JSONSchema/JSONSchemaTemplate/WorkflowArrayFieldTemplate.tsx

```diff
@@ -53,6 +57,57 @@ const WorkflowArrayFieldTemplate = (props: FieldProps) => {
     ? t('message.filter-pattern-placeholder')
     : '';
   const options = generateOptions();
+  const { onCopyToClipBoard, onPasteFromClipBoard, hasCopied } = useClipboard(
+    JSON.stringify(value)
+  );
+
+  const handleCopy = useCallback(
+    async (e: React.MouseEvent) => {
+      e.stopPropagation();
+      await onCopyToClipBoard();
+    },
+    [value, onCopyToClipBoard]
+  );
+
+  const handlePaste = useCallback(async () => {
+    const text = await onPasteFromClipBoard();
+    if (!text) {
+      return;
+    }
+
+    let processedValues: string[] = [];
```

Comment:
What if paste value is not string?

Reply:
Even if the input contains numbers, splitCSV will always return strings